sponsors.html

```diff
@@ -62,7 +62,8 @@
 
 		  <!-- Three columns of text below the carousel -->
 		  <div class="row">
-				<p>Thank you to our wonderful sponsors for supporting Omaha Girls Who Code</p>
+				<h1>Sponsors -- we couldn't do it without you!</h1>
```

Comment:
I know Shonna is doing the sponsor page. I just had a random thought about the Volunteer/Sponsor header messages. We don't have to keep it.